release/apparmor.go

```diff
@@ -133,3 +143,48 @@ func AppArmorFeatures() []string {
 	}
 	return appArmorFeatures
 }
+
+// parser probe related code
+type apparmorParserFeature struct {
+	feature string
+	rule    string
+}
+
+var requestedParserFeatures = []apparmorParserFeature{
+	{"unsafe", "change_profile unsafe /**,"},
+}
+
+// tryParser will run the parser on the rule to determine if the feature is
+// supported.
+func tryParser(rule string) bool {
+	cmd := exec.Command("apparmor_parser", "-p")
```

Comment:
What will happen when `apparmor_parser` is not on PATH? Like on SELinux distributions? Specifically I think we should just return false without any errors.

Reply:
I didn't trace it all down, but spread wasn't grumpy about it. I added a LookupPath just in case just now.